model/riscv_sys_control.sail

```diff
@@ -17,65 +17,65 @@ function csrPriv(csr : csreg) -> priv_level = csr[9..8]
 function is_CSR_defined (csr : csreg, p : Privilege) -> bool =
```

Comment:
I think we should also be able to remove the `p` parameter here and remove it from `ext_is_CSR_defined` as well

Reply:
Ah good call, I'll do that.

Reply:
Done